docs/codeql/codeql-cli/creating-codeql-databases.rst

```diff
@@ -17,6 +17,10 @@ Before you generate a CodeQL database, you need to:
 - Check out the version of your codebase you want to analyze. The directory
   should be ready to build, with all dependencies already installed.
 
+For information about using the CodeQL CLI in a CI system to create results
```

Comment:
Should we specifically mention "aside from GitHub Actions"?

Reply:
Good point - we probably should be explicit here.

Reply:
```suggestion
For information about using the CodeQL CLI in an third-party CI system to create results
```